index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formulário</title>
+</head>
+<body>
+    <h1>Time de Valorant</h1>
+    <p>Formulario para montar um time de valorant</p>
+    <form action="mailto:alefstarkin@gmail.com">
+        <div>
+            <label>Nome:</label><input type="text" placeholder="Digite seu nome" required>
+            <p></p>
+            <label>Email: </label><input type="email" placeholder="exemplo@email.com" required>
+            <p></p>
+            <label>Joga com quantos agentes?: </label><input type="number" min="1" max="5" required>
+            <p></p>
+            <label>Qual a sua role? </label>
+                <select>
+                    <option>Duelista</option>
```

Comment:
Os elementos `option` precisam ter um `value` para que o elemento `form` saíba qual é o valor que o usuário escolheu. O conteúdo do elemento `option` é meramente visual.
Então, no seu caso, poderia ser algo como:

```suggestion
                    <option value="duelista">Duelista</option>
```